mmcv/cnn/bricks/crf.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from https://github.com/kmkurn/pytorch-crf/blob/master/torchcrf
+import torch
+import torch.nn as nn
+from typing import List, Optional
+
+
+class ConditionalRandomField(nn.Module):
+    """Conditional random field.
+    This module implements a conditional random field [LMP01]_.
```

Comment:
```suggestion
    """Conditional random field.
    
    This module implements a conditional random field [LMP01]_.
```

Reply:
Refer to the correct citation format in https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/models/backbones/resnet.py#L311-L386.